data/human/free worlds 1 start.txt

```diff
@@ -1308,25 +1326,10 @@ mission "FW Pirates: Attack 2"
 	destination "Greenrock"
 	clearance
 	to offer
-		has "FW Pirates: Attack 1: done"
+		has "event: battle for Greenrock"
```

Comment:
Use the condition we set before, and leave the mission condition in for clarity.
```suggestion
		has "FW Pirates: Attack 1: done"
		has "fw: will attack greenrock"
```